Binary files orig/archive/src/.Makevars.ucrt.swp and patched/archive/src/.Makevars.ucrt.swp differ
diff -Nru orig/archive/src/Makevars.ucrt patched/archive/src/Makevars.ucrt
--- orig/archive/src/Makevars.ucrt	2024-04-28 11:32:46.000000000 +0200
+++ patched/archive/src/Makevars.ucrt	2024-10-16 19:11:53.098794100 +0200
@@ -1,7 +1,11 @@
 PKG_CPPFLAGS= -I. -Icpp11/include
 
-PKG_LIBS = -larchive -lxml2 -lcrypto -lnettle -lregex -lexpat -llzo2 \
-	   -llzma -llz4 -lbz2 -lz -lzstd -liconv -lws2_32
+ifeq (,$(shell pkg-config --version 2>/dev/null))
+    PKG_LIBS = -larchive -lxml2 -lcrypto -lnettle -lregex -lexpat -llzo2 \
+               -llzma -llz4 -lbz2 -lz -lzstd -liconv -lws2_32 -lbcrypt
+else
+    PKG_LIBS = $(shell pkg-config --libs libarchive)
+endif 
 
 LIB_CON = ../inst/lib$(R_ARCH)/libconnection.dll
 
